docs/getting_started/setup_long_polling.md

```diff
@@ -0,0 +1,30 @@
+# Setup Long polling Publisher
+
+This document walks through setting up a publisher using the long polling technique.
+
+## Configure `WAGTAIL_LIVE_PUBLISHER`
+
+In order to use the long polling technique for the publishing part, add this to your `settings`:
+```python
+WAGTAIL_LIVE_PUBLISHER = "wagtail_live.publishers.LongPollingPublisher"
+```
+
+You can optionally set a polling timeout by doing so:
+```python
+WAGTAIL_LIVE_POLLING_TIMEOUT = value # in seconds
+```
+The default value is **30**(s).
+
+## Add publisher template
+
+We also need to add this to our `live_blog_page.html` template:
+```python
+{% include "wagtail_live/polling/long_polling.html" %}
+```
+
+Make sure you have these 2 lines in your template:
```

Comment:
Why are we are telling them about `wagtail_live/polling/long_polling.html` twice?

Reply:
To make sure that not only `wagtail_live/polling/long_polling.html` should be included but also `wagtail_live/live_posts.html`. I can remove it if it's not necessary.